platform/openide.util.ui/test/unit/src/org/openide/util/UtilitiesTest.java

```diff
@@ -320,12 +322,15 @@ public Toolkit getToolkit() {
             return customToolkit;
         }
     }
-    
+
+    /* 2023-12-02: Various methods have been commented out as they can no longer be overridden on
+                   Java 9. All of these simply used to throw a "not implemented" exception.  */
```

Comment:
please remove the commented out bits and this comment too.

If the module is only loadable on 11+, this information isn't important anymore and only adds to the noise. Git keeps track of the file history so the removed code isn't lost.